packages/nimble-components/src/drawer/styles.ts

```diff
@@ -88,8 +91,9 @@ export const styles = css`
     }
 
     :host([location='left']) .dialog-contents {
-        border-right: ${borderWidth} solid ${popupBoxShadowColor};
-        box-shadow: 3px 0px 8px ${popupBoxShadowColor};
+        border-right: ${borderWidth} solid ${elevation3BoxShadowColor};
+        box-shadow: ${drawerBoxShadowOffsetX} ${elevation3BoxShadowOffsetY}
```

Comment:
commentary / venting: Frustrating that box-shadow is not a shorthand property and this kinda forces us to break-out all the parts :( 

Looks like they are coalescing on breaking out the parts in the future though, maybe in a few years we can remove a bunch of these tokens: https://github.com/w3c/csswg-drafts/issues/4431